contracts/libraries/SafeToL2Migration.sol

```diff
@@ -95,12 +91,12 @@ contract SafeToL2Migration is SafeStorage {
             0,
             0,
             0,
-            address(0),
+            payable(address(0)),
             address(0),
```

Comment:
I think this is the wrong address. This is the token and not the refund receiver.

Reply:
Apologies 🤦🏾 Updated 👍🏾